Refactored: Code

pritam-kr opened the pull request.

pritam-kr committed:
Fetched wishlist initial data

pritam-kr committed:
Changing in delivery charge

pritam-kr committed:
Code refactored

saifudinbadani commented on src/component/FeaturedProductCard/FeaturedProductCard.jsx:
```diff
@@ -32,7 +34,7 @@ const FeaturedCardProduct = ({eachProduct}) => {
 
       <div className="btn-wishlist btn-wishlist-icon2">
-        {wishlist.find((eachItem) => eachItem._id === eachProduct._id) ? (<FaIcons.FaHeart className="wishlist-icon" style={{color: "#cc2e2e"}} onClick={() =>  {removeFromWishlist(eachProduct)}} />) : (<FaIcons.FaHeart className="wishlist-icon" onClick={() => addToWishlist(eachProduct)}/>)}
+        {wishlist.find((eachItem) => eachItem._id === eachProduct._id) ? (<FaIcons.FaHeart className="wishlist-icon" style={{color: "#cc2e2e"}} onClick={() =>  {removeFromWishlist(eachProduct)}} />) : (<FaIcons.FaHeart className="wishlist-icon" onClick={() => (token ? addToWishlist(eachProduct) : navigate("/login"))}/>)}
```
This is part is simple and very well written.

saifudinbadani reviewed (approved):
Found your code very easy to read.

pritam-kr committed:
Implemented Search filter based on onChangeEvent

pritam-kr committed:
Implemented: Category Features

If I click on any one of the categories I will be redirected to the product list page with the selected category.

pritam-kr committed:
Link updated
